### ISSUE TITLE
Model class and pre-trained checkpoints

### ISSUE THREAD

**Issue by theodpzz** [wasserth/TotalSegmentator#263]
Hello,

New to TotalSegmentator.

I'm looking for the script with the torch class of TotalSegmentator which can take as input the CT scan, and outputs masks of each organ.

I'm also looking for pre-trained weights of this model.

Please, could you help me to find it ?

Thank you !

**Comment by wasserth** [comment]
TotalSegmentator is using nnU-Net. The torch class is inside of nnU-Net. You will have investigate that library to find the torch class: https://github.com/MIC-DKFZ/nnUNet. 
The pretrained weights are in the releases: https://github.com/wasserth/TotalSegmentator/releases
